sys/test_utils/result_output/json/result_output_json.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    test_utils_result_output_json Test result output with JSON
+ * @brief       Enable this module to have results output as json
+ * @ingroup     test_utils_result_output
+ * @{
+ *
+ * @file
+ * @brief       Result output in json
+ *
+ *
+ * @author      Kevin Weiss <kevin.weiss@haw-hamburg.de>
+ * @}
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "fmt.h"
+
+#include "test_utils/result_output.h"
+
+void turo_container_open(turo_t *ctx)
+{
+    (void)ctx;
+    print_str("[");
+}
+
+void turo_s32(turo_t *ctx, int32_t val)
+{
+    (void)ctx;
+    print_s32_dec(val);
+    print_str(",");
```

Comment:
So every turo json container has a comma at the end? `[ 1253, ]`?

Reply:
Yup, the rapidjson parser among others allow for trailing commas and this means we do not have to keep states.

Reply:
or have different functions for print arrays and such.

Reply:
but it is non-standard. my vim cries all red on me if a there's a wrong comma.
"Be conservative in what you send, be liberal in what you accept".

Would it be a lot of work to add that state?